kolibri/core/assets/src/vue/nav-bar/index.vue

```diff
@@ -14,9 +14,17 @@
         <svg class="nav-icon" src="../icons/coach.svg"/>
         <div class="label">{{ $tr('coach') }}</div>
       </nav-bar-item>
-      <nav-bar-item v-if="isAdminOrSuperuser" href="/management" :active="manageActive">
+      <nav-bar-item v-if="loggedIn" href="/management" :active="profileActive">
         <svg class="nav-icon" src="../icons/manage.svg"/>
-        <div class="label">{{ $tr('manage') }}</div>
+        <div class="label">{{ $tr('profile') }}</div>
+      </nav-bar-item>
+      <nav-bar-item v-if="loggedIn">
+        <svg class="nav-icon" src="../icons/manage.svg"/>
+        <div class="label">{{ $tr('logOut') }}</div>
+      </nav-bar-item>
+      <nav-bar-item v-else href="/signin">
```

Comment:
Unconcerned with whether or not it's active because the navbar isn't showing anyway